suite-native/firmware/src/screens/FirmwareUpdateInProgressScreen.tsx

```diff
@@ -0,0 +1,191 @@
+import { useCallback, useEffect } from 'react';
+import { Linking } from 'react-native';
+import Animated, { FadeInDown, FadeOutDown, LinearTransition } from 'react-native-reanimated';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+import { useDispatch } from 'react-redux';
+
+import { useNavigation } from '@react-navigation/native';
+
+import { authorizeDeviceThunk } from '@suite-common/wallet-core';
+import { Box, Button, Text, VStack } from '@suite-native/atoms';
+import { ConfirmOnTrezorImage, setDeviceForceRememberedThunk } from '@suite-native/device';
+import { requestPrioritizedDeviceAccess } from '@suite-native/device-mutex';
+import { Translation } from '@suite-native/intl';
+import {
+    DeviceSettingsStackParamList,
+    DeviceStackRoutes,
+    Screen,
+    StackNavigationProps,
+} from '@suite-native/navigation';
+import TrezorConnect from '@trezor/connect';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { UpdateProgressIndicator } from '../components/UpdateProgressIndicator';
+import { useFirmware } from '../hooks/useFirmware';
+type NavigationProp = StackNavigationProps<
+    DeviceSettingsStackParamList,
+    DeviceStackRoutes.FirmwareUpdateInProgress
+>;
+
+const bottomButtonsContainerStyle = prepareNativeStyle<{ bottom: number }>((utils, { bottom }) => ({
+    position: 'absolute',
+    left: utils.spacings.sp16,
+    right: utils.spacings.sp16,
+    bottom,
+}));
+
+export const FirmwareUpdateInProgressScreen = () => {
+    const dispatch = useDispatch();
+    const { applyStyle } = useNativeStyles();
+    const navigation = useNavigation<NavigationProp>();
+    const { bottom: bottomSafeAreaInset } = useSafeAreaInsets();
+    const {
+        operation,
+        setIsFirmwareInstallationRunning,
+        confirmOnDevice,
+        firmwareUpdate,
+        progress,
+        status,
+        resetReducer,
+        translatedText,
+        mayBeStucked,
+    } = useFirmware({});
```

Comment:
```suggestion
    } = useFirmware();
```

Please set `params = {}` as default value in native `useFirmware`.